build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-multiplatform.gradle.kts

```diff
@@ -198,5 +199,10 @@ fun androidTargets() = listOf(
 )
 
 rootProject.the<NodeJsRootExtension>().apply {
-    nodeVersion = "21.0.0"
+    nodeVersion = "21.0.0-v8-canary202310177990572111"
+    nodeDownloadBaseUrl = "https://nodejs.org/download/v8-canary"
```

Comment:
Is the canary version required for WASM support?

Reply:
Yes, it is: WASM requires v8 >= 11.9 and it is bundled only in canary builds.